spring-cloud-modules/spring-cloud-openfeign/src/main/java/com/baeldung/cloud/openfeign/customizederrorhandling/config/FeignConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.cloud.openfeign.customizederrorhandling.config;
+
+import feign.Logger;
+import feign.codec.ErrorDecoder;
+import org.springframework.context.annotation.Bean;
+
+public class FeignConfig {
```

Comment:
Should this have `@Configuration`?

Reply:
It's not required as the class is passed to feign. And, this Spring configuration is not needed for that.